conmon-rs/common/proto/conmon.capnp

```diff
@@ -20,6 +20,7 @@ interface Conmon {
         bundlePath @1 :Text;
         terminal @2 :Bool;
         exitPaths @3 :List(Text);
+        logPath @4 :Text;
```

Comment:
Eventually, we may need to support multiple log driver/log path pairs. currently, conmon just has them specified as `log_driver:/path`, but I think we could do better. maybe we could create a LogConfig and have a vector of those LogConfigs in the request?

Reply:
Good, following up on this in https://github.com/containers/conmon-rs/pull/315